### PR TITLE
Bug 1756251 Correct Registry Operator content

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1756251

### CODE REVIEW

File: modules/registry-operator-config-resources-configmap.adoc [hunk 1]
```diff
@@ -107,4 +107,4 @@ data:
     -----END CERTIFICATE-----
 ----
 <1>  If the registry has the port, such as `registry-with-port.example.com:5000`,
-`:` should be replaced with `...`.
+`:` should be replaced with `. . .`.
```
Comment (xiuwang) [inline modules/registry-operator-config-resources-configmap.adoc]: The dot connector should like here, `image-registry.openshift-image-registry.svc..5000`

File: modules/registry-operator-configuration-resource-overview.adoc [hunk 2]
```diff
@@ -36,6 +36,9 @@ and upstream registries.
 |`Storagetype`: Details for configuring registry storage, for example S3 bucket
 coordinates. Normally configured by default.
 
+|`readOnly`
```
Comment (xiuwang) [inline modules/registry-operator-configuration-resource-overview.adoc]: It's better to keep consistent with other parameters, start with upper case letter.

**Comment by xiuwang** [comment]
LGTM, thanks

**Comment by bmcelvee** [comment]
Thanks so much!

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review. This one is tiny.

**Comment by bmcelvee** [comment]
This is a tiny PR with very minor changes. I'm going ahead with the merge and cherrypick.

**Pull request merged**